src/move/move.ts

```diff
@@ -0,0 +1,63 @@
+import { Board } from "../board/board";
+import { GameState } from "../game/state";
+import { NEUTRALE_TILE, findTile, findTileByCoordinate } from "../tile/tile";
+
+interface Action {
+  value: any;
+}
```

Comment:
FMI why did you have to use `any`?
Also, since the type is really not informative about what you are storing in `value`, can you find a better name?

Reply:
It's the result of Prompts library: `{value: {...}}`

Reply:
Interface Action is now more discriminating